code/controllers/subsystems/ticker.dm

```diff
@@ -112,6 +113,9 @@ SUBSYSTEM_DEF(ticker)
 		return
 	if(round_progressing && last_fire)
 		pregame_timeleft -= world.time - last_fire
+	if(pregame_timeleft <= 300 && !tipped)
+		send_random_tip()
+		tipped = 1
```

Comment:
Minor nit-picks. If `tipped` is intended to be used as a boolean, can we keep that formatting? It would be good for readability to use the `SECONDS` marco here
```suggestion
	if(pregame_timeleft <= 30 SECONDS && !tipped)
		send_random_tip()
		tipped = TRUE
```